### DISCUSSION TITLE
Feature request: cards edit mode

### DISCUSSION THREAD

**Discussion by jgroenen** [josdejong/svelte-jsoneditor#224] [Ideas]
Using json editor online as a go-to tool for json stuff, would love to use it as a go-to tool for "cms"-like stuff too

![proposal-json-online-editor](https://user-images.githubusercontent.com/1716809/218512483-eb1cd327-c827-4981-b845-eace05915b86.png)

**Comment by jgroenen** [comment]
![proposal-json-online-editor](https://user-images.githubusercontent.com/1716809/218562848-220b0084-692b-4213-9df6-76496e49f2cd.png)
![Fo2927gXgAEi_Fb](https://user-images.githubusercontent.com/1716809/218562851-26f24da9-81d0-41d0-8135-dc282736e8e7.png)

**Comment by josdejong** [comment]
Thanks for your suggestion Johan. It's an interesting idea to introduce a new mode `card` which populates fields based on a JSON schema.

I'll move this idea from "Issues" into the "Discussions" > "Ideas" section.

**Comment by AlexRMU** [comment]
You can add the "expand row" button to the table mode. It will be located at the end of the line or in the context menu. It will create a container under the row, with a width equal to the row but no more than 100% of the editor's size (to look like a modal). In this container, the row fields will be arranged vertically, like in tree mode, but in the form of a table.
For example:
![111](https://github.com/user-attachments/assets/62c9d1d2-5fcd-42e8-84a9-17d20ccec28b)
